src/components/draggable/draggable.spec.tsx

```diff
@@ -190,9 +171,9 @@ describe("Draggable", () => {
     });
 
     it("should drag items within container 1", () => {
-      const container1 = mountNode.querySelector("#container-1");
+      const container1 = mountNode?.querySelector("#container-1");
 
-      const draggableItems = getDraggableItems(container1);
+      const draggableItems = getDraggableItems(container1 as never);
```

Comment:
If you change `getDraggableItems` to the following you can remove `as never` wherever you've used it

```
const getDraggableItems = (mountNode: Element | null) =>{
    const draggableItems = mountNode?.querySelectorAll('div[data-element="draggable"]')
    return draggableItems ? Array.from(draggableItems) : [];
  }
```